### PR TITLE
Add remote-control-with-oculusvr for Clover

### PR DESCRIPTION
**Pull request by GalinaDa**

### CODE REVIEW

**Comment by okalachev** [comment]
Is this a CopterHack article? What is the team?

**Comment by GalinaDa** [comment]
Да, статья для CopterHack, команда - ProCleVeR18.03.2021, 20:34, "Oleg Kalachev" ***@***.***>:
Is this a CopterHack article? What is the team?

**Comment by okalachev** [comment]
What is the team name? And, please, give some contact (e-mail, Telegram, ...) you want to put into the article.

File: docs/ru/remote-control-with-oculusvr.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Для отправки используем:
+```csharp
+socketClient.Send(Encoding.ASCII.GetBytes("1"));
+```
+Для  принятия  данных  используем:
+```csharp
+socketClient.Receive(Encoding.ASCII.GetBytes("1024"));
+```
+
+![Видео демонстарция работоспсобности результата](https://user-images.githubusercontent.com/60854798/111510627-8c0a9700-8780-11eb-8627-eeaae06612f8.mp4)
```
Comment (okalachev) [inline docs/ru/remote-control-with-oculusvr.md:73]: Please, put the videos on YouTube and embed into the article like this:

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/HdHbZFz7nR0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by GalinaDa** [comment]
> Please, give some contact (e-mail, Telegram, ...) you want to put into the arti

**Comment by okalachev** [comment]
I don't need to close the PR to change something, you just need to push new changes to your branch.

**Comment by GalinaDa** [comment]
> Please, give some contact (e-mail, Telegram, ...) you want to put into the article.

galyadavydenko@yandex.ru

**Comment by GalinaDa** [comment]
I pushed the latest article changes to my fork according to recommendations

**Comment by GalinaDa** [comment]
> 
> 
> I don't need to close the PR in order change something, you just need to push new changes to your branch.

I pushed the latest article changes to my fork according to recommendations

**Pull request merged**